avalon/tools/models.py

```diff
@@ -241,7 +243,7 @@ def set_assets(self, asset_ids):
             item = Item({
                 "name": "No task",
                 "count": 0,
-                "icon": default_icon,
+                "icon": self._icons["__no_task__"],
                 "enabled": False,
```

Comment:
Could we make the `self._icons["__no_task__"]` be unified to `no_task_icon` ?
Like the `default_icon` above, for better reading :relaxed:

Reply:
Are you saying you want me to solely declare a variable above it? :)
```python
            no_task_icon = self._icons["__no_task__"]
            item = Item({
                "name": "No task",
                "count": 0,
                "icon": no_task_icon,
                "enabled": False,
            })
```

Like that?

Reply:
Yes ! And if that make sense to you :relaxed:

Reply:
Fine with me. Pushed it https://github.com/getavalon/core/pull/452/commits/a13433b3424d1f628af3127ae5a92c6682467849 :)